contents/docs/data-warehouse/views.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Creating views
+sidebar: Docs
+showTitle: true
+availability:
+    free: full
+    selfServe: full
+    enterprise: full
+---
+
+The PostHog data warehouse supports views. With views, you can save your most used queries and reference them in subsequent queries.
+
+### Creating a view
+
+Query views are created directly inside the HogQL editor. If the query has valid view characteristics, the "Save as view" button will be enabled. You will be prompted to give the view a name which will be the name that can be refeneced in other queries. For a query to be a valid view, all fields being accessed must be aliased. The alias names will be how you access the fields on the view. 
+
+> **Note:** Currently, nested views are not supported. You cannot create a view based on other views only base PostHog models.
+
+![valid view](../../images/features/data-warehouse/valid-view.png)
+
+### Extending PostHog models with views
```

Comment:
Would this only work if my view has the same primary key as my table? 
E.g. If my stripe customer data table has a customer_id primary key, then my view (with product usage data) also needs to be defined in a way so that it only has 1 row per customer_id?

Reply:
For that use case, yep you would ideally have a view where the customer_id/user_id is distinct.